app/design/adminhtml/default/default/template/catalog/product/edit/price/group.phtml

```diff
@@ -52,13 +52,13 @@ $_showWebsite= $this->isMultiWebsites();
                 <th class="last"><?php echo Mage::helper('catalog')->__('Action'); ?></th>
             </tr>
         </thead>
-        <tbody id="<?php echo $_htmlId; ?>_container"></tbody>
         <tfoot>
             <tr>
                 <td <?php if (!$_showWebsite): ?>style="display: none;"<?php endif; ?>></td>
                 <td colspan="4" class="a-right"><?php echo $this->getAddButtonHtml(); ?></td>
             </tr>
         </tfoot>
+        <tbody id="<?php echo $_htmlId ?>_container"></tbody>
```

Comment:
Why is `tbody` below `tfoot`?

Reply:
I moved it because the [HTML Validator](http://users.skynet.be/mgueury/mozilla) Firefox plugin ask this change.